### DISCUSSION TITLE
Migration from tonesto7 plugin

### DISCUSSION THREAD

**Discussion by DMKultra** [iklein99/homebridge-smartthings#117] [General]
Looking for tips and tricks migrating from the old smarthings plug in.  Is it as simple as removing the old and configuring the new one?  Anything I need to look out for?  
Thanks in advance!

**Comment by iklein99** [comment]
That’s pretty much it.  This plugin will find all of the supported devices - note not everything is supported. 

I’d disable the old one before removing it, then add this one and try it.

**Reply by DMKultra** [reply]
Thanks worked like a charm!  Thanks for the plugin!

**Comment by dalejake** [comment]
what a brillant repalcement for the smartapp v2
